lib/domain/entities/item.dart

```diff
@@ -0,0 +1,13 @@
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+
+class Item {
+  const Item(
+      {@required this.name, @required this.image, @required this.category, @required this.effect});
```

Comment:
```suggestion
      {@required this.name, @required this.image, @required this.category, @required this.effect,});
```
Please add a  trailing `,` to make the formatter works correctly